pkg/runtime/runtime.go

```diff
@@ -1928,7 +1939,7 @@ func (a *DaprRuntime) publishMessageHTTP(ctx context.Context, msg *pubsubSubscri
 
 	if err != nil {
 		diag.DefaultComponentMonitoring.PubsubIngressEvent(ctx, msg.pubsub, strings.ToLower(string(pubsub.Retry)), msg.topic, elapsed)
-		return errors.Wrap(err, "error from app channel while sending pub/sub event to app")
+		return fmt.Errorf("%w from app channel while sending pub/sub event to app: %v", ErrRetriable, err)
```

Comment:
Not sure if I should handle it this way, do I need to use multierror to wrap multiple errors at the same time?